sycl/test/sub_group/vote.cpp

```diff
@@ -25,19 +25,29 @@ void check(queue Queue, const int G, const int L, const int D, const int R) {
     buffer<int, 1> sganybuf(G);
     buffer<int, 1> sgallbuf(G);
 
+    // Initialise buffer with zeros
+    Queue.submit([&](handler &cgh) {
+      auto sganyacc = sganybuf.get_access<access::mode::read_write>(cgh);
+      auto sgallacc = sgallbuf.get_access<access::mode::read_write>(cgh);
+      cgh.parallel_for<class init>(range<1>{(unsigned)G}, [=](id<1> index) {
```

Comment:
Is `(unsigned)` required?

Reply:
Yes. It might be a bug, but the test fails to compile otherwise. Will take a look later.

Reply:
It's bug in range constructor indeed. Will be fixed later.